wallets/lnc/client.js

```diff
@@ -15,20 +13,12 @@ export async function testSendPayment (credentials, { logger }) {
 }
 
 export async function sendPayment (bolt11, credentials, { logger }) {
-  const hash = bolt11Tags(bolt11).payment_hash
   return await mutex.runExclusive(async () => {
-    try {
-      const lnc = await getLNC(credentials, { logger })
-      const { paymentError, paymentPreimage: preimage } = await lnc.lnd.lightning.sendPaymentSync({ payment_request: bolt11 })
-      if (paymentError) throw new Error(paymentError)
-      if (!preimage) throw new Error('No preimage in response')
-      return preimage
-    } catch (err) {
-      const msg = err.message || err.toString?.()
-      if (msg.includes('invoice expired')) throw new InvoiceExpiredError(hash)
-      if (msg.includes('canceled')) throw new InvoiceCanceledError(hash)
-      throw err
-    }
+    const lnc = await getLNC(credentials, { logger })
+    const { paymentError, paymentPreimage: preimage } = await lnc.lnd.lightning.sendPaymentSync({ payment_request: bolt11 })
+    if (paymentError) throw new Error(paymentError)
+    if (!preimage) throw new Error('No preimage in response')
+    return preimage
```

Comment:
I removed the `try/catch` here since we already have error handling in `waitForPayment` (which also checks for canceled or expired invoices) and in `sendPayment` which wrapps `wallet.def.sendPayment`.

I think this was just old code that we no longer need.